infra/services/aiSearch-post.ps1

```diff
@@ -0,0 +1,181 @@
+param(
+    [string] $ResourceGroupName,
+    [string] $BaseName
+)
+
+$ErrorActionPreference = "Stop"
+
+. "$PSScriptRoot/../../eng/common/scripts/common.ps1"
+
+# Get current subscription ID
+$subscriptionId = (Get-AzContext).Subscription.Id
+
+$apiVersion = '2020-06-30'
+$token = Get-AzAccessToken -ResourceUrl https://search.azure.com -AsSecureString | Select-Object -ExpandProperty Token
+$uri = "https://$BaseName.search.windows.net"
+
+$indexDefinition = @{
+  name = "products"
+  fields = @(
+    @{ name = "chunk_id"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true; sortable = $true; key = $true; analyzer = "keyword" }
+    @{ name = "parent_id"; type = "Edm.String"; filterable = $true; retrievable = $true; stored = $true }
+    @{ name = "chunk"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true }
+    @{ name = "title"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true }
+    @{ name = "header_1"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true }
+    @{ name = "header_2"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true }
+    @{ name = "header_3"; type = "Edm.String"; searchable = $true; retrievable = $true; stored = $true }
+    @{ name = "text_vector"; type = "Collection(Edm.Single)"; searchable = $true; retrievable = $true; stored = $true; dimensions = 1536; vectorSearchProfile = "products-azureOpenAi-text-profile" }
+    @{ name = "category"; type = "Edm.String"; searchable = $true; filterable = $true; retrievable = $true; stored = $true; facetable = $true }
```

Comment:
Let's set `searchable = $false` for this category field, my bad that I had it set to true in my example.